spec/app_spec.rb

```diff
@@ -25,7 +25,14 @@
   end
 
   it "should get urllib3 both from channel and not" do
-    get "/packages/pkgs%252Fmain/urllib3"
+    get "/package"
+    expect(last_response).to be_ok
+    json = JSON.parse(last_response.body)
+    expect(json["error"]).to eq "Please provide at least a package name ?name= parameter"
+  end
+
+  it "should get urllib3 both from channel and not" do
```

Comment:
I think there should also be a spec to check that the default channel is `pkgs/main`.

Reply:
There is on line 71ish